File: JavaScript-Solutions/return-largest-number-in-array-of-arrays.js
function largestInArrayOfArrays(arr) {
  var largest = [];
  for (var i = 0; i < arr.length; i++) {
    var maxNum = arr[i][0];
    for (var j = 1; j < arr[i].length; j++) {
      if (maxNum < arr[i][j]) {
        maxNum = arr[i][j];
      }
    }
    largest.push(maxNum);
  }
  
    
  return largest;
}

File: Java-Solutions/return-largest-number-in-array-of-arrays.java
package largestnumbersinarrays;
public class LargestNumbersInArrays {

    public static void main(String[] args) {
    }

    public static int[] largestNumbersInArrays(int[][] arr) {
        int[] largest = new int[arr.length];
        int i = 0;
        for (int[] subArr : arr) {
            int maxNum = subArr[0];
            for (int num : subArr) {
                if (maxNum < num) {
                    maxNum = num;
                }
            }
            largest[i] = maxNum;
            i++;
        }

        return largest;
    }
}
